pandas/compat/__init__.py

```diff
@@ -118,3 +119,8 @@ def get_lzma_file(lzma):
             "might be required to solve this issue."
         )
     return lzma.LZMAFile
+
+
+__all__ = [
+    "is_numpy_dev",
```

Comment:
dont we also need everything else currently in the namespace?

Reply:
Yeah probably, but not sure if we should rather do this as a follow up? If numpy fixes the bug or after we have fixed this on our site, we could simply revert this pr to avoid touching 30 files. If we do more in here, this may get tricky